packages/suite/src/views/wallet/send/components/Outputs/components/Amount/components/Fiat.tsx

```diff
@@ -74,7 +74,7 @@ export const Fiat = ({ output, outputId, labelHoverRight, labelRight }: FiatProp
 
     const currencyValue = watch(currencyInputName);
 
-    const recalculateFiat = (rate: number) => {
+    const recalculateFiat = (rate?: number) => {
```

Comment:
I would leave `rate: number` 

and just make check before this method is called (it is called just on 1 place)

Reply:
a1718ae7de5e8b020f8b1c9b763eee8039cc6bd1